packages/block-editor/src/components/block-popover/inbetween.js

```diff
@@ -91,66 +91,81 @@ function BlockPopoverInbetween( {
 		};
 	}, [ previousElement, nextElement, isVertical ] );
 
-	const getAnchorRect = useCallback( () => {
+	const popoverAnchor = useMemo( () => {
 		if ( ( ! previousElement && ! nextElement ) || ! isVisible ) {
-			return {};
```

Comment:
Change #1: returning `undefined` since an empty object wouldn't make sense for the `anchor` prop